atsign-foundation/noports
# test: Unit tests for Sshrvd Channel

**- What I did**

- See my comments in the Files changed tab

General things I did:
- Removed dartLocal client
- Added notificationService.subscribe to be a member of AtClientBindings just like notify
- Started tests for the initial tunnel formation (stopped, since I realized it wasn't the best place to start)
- Tests for Sshrvd Channel

**- How I did it**

**- How to verify it**

- Ensure `unit_tests / noports_core-unit_tests (pull_request)` passes
- Ensure that you are happy with what and how I am testing everything

**- Description for the changelog**
test: Unit tests for Sshrvd Channel
